packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/example.html

```diff
@@ -66,6 +66,26 @@ <h2>Examples</h2>
       <span slot="help">This is a help message</span>
     </gux-form-field-color>
   </fieldset>
+
+  <fieldset>
+    <legend>Label Info</legend>
+
+    <gux-form-field-color>
+      <input slot="input" type="color" name="e-1" value="#cc3ebe" />
+      <label slot="label">Default</label>
+      <gux-label-info-beta slot="label-info">
+        <span slot="content">This is some tooltip text</span>
+      </gux-label-info-beta>
+    </gux-form-field-color>
+
+    <gux-form-field-color>
+      <input slot="input" type="color" name="e-1" value="#cc3ebe" required />
+      <label slot="label">Default</label>
+      <gux-label-info-beta slot="label-info">
+        <span slot="content">This is some tooltip text</span>
+      </gux-label-info-beta>
+    </gux-form-field-color>
+  </fieldset>
```

Comment:
Added this just to make it easier to verify the way it interacts with required. Will remove once review it done.